packages/language-server/src/nodeServer.ts

```diff
@@ -22,35 +29,78 @@ connection.onInitialize((params) => {
 
 	const { typescript, diagnosticMessages } = loadTsdkByPath(tsdk, params.locale);
 
+	contentIntellisenseEnabled = params.initializationOptions?.contentIntellisense ?? false;
+	let collectionConfigs: { folder: URI; config: CollectionConfig['config'] }[] = [];
+
+	if (contentIntellisenseEnabled) {
+		// The vast majority of clients support workspaceFolders, but notably our tests currently don't
+		// Ref: https://github.com/volarjs/volar.js/issues/229
+		const folders =
+			params.workspaceFolders ?? (params.rootUri ? [{ uri: params.rootUri }] : []) ?? [];
```

Comment:
This brought me back to 2 years ago when you actually needed to handle `rootURI`. Good times (no)